tests/testthat/_snaps/simulate-infections.md

```diff
@@ -2,67 +2,67 @@
 
               variable       date     value
                 <char>     <Date>     <num>
-     1:     infections 2023-01-01 120.00000
-     2:     infections 2023-01-02 144.00000
-     3:     infections 2023-01-03 172.80000
-     4:     infections 2023-01-04 207.36000
-     5:     infections 2023-01-05 248.83200
-     6:     infections 2023-01-06 298.59840
-     7:     infections 2023-01-07 358.31808
-     8:     infections 2023-01-08 286.65446
-     9:     infections 2023-01-09 229.32357
-    10:     infections 2023-01-10 183.45886
-    11:     infections 2023-01-11 146.76709
-    12:     infections 2023-01-12 117.41367
-    13:     infections 2023-01-13  93.93093
-    14:     infections 2023-01-14  75.14475
-    15: reported_cases 2023-01-01 125.00000
-    16: reported_cases 2023-01-02 135.00000
-    17: reported_cases 2023-01-03 195.00000
-    18: reported_cases 2023-01-04 224.00000
-    19: reported_cases 2023-01-05 253.00000
-    20: reported_cases 2023-01-06 328.00000
-    21: reported_cases 2023-01-07 364.00000
-    22: reported_cases 2023-01-08 278.00000
-    23: reported_cases 2023-01-09 206.00000
-    24: reported_cases 2023-01-10 169.00000
-    25: reported_cases 2023-01-11 144.00000
-    26: reported_cases 2023-01-12 109.00000
-    27: reported_cases 2023-01-13  80.00000
-    28: reported_cases 2023-01-14  95.00000
+     1:     infections 2023-01-01 100.00000
```

Comment:
has something changed in the date indexing here?

Reply:
to be revisited once simulation code has been updated